content/blog/2022/2022-06-01-blog-minions-new.md

```diff
@@ -0,0 +1,63 @@
++++
+author = "Zala Gruden"
+date = "2022-06-02"
+draft = false
+title = "Teaching fourth-graders data science"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-06-02-hands-up.jpg"
+frontPageImage = "/blog_img/2022/2022-06-02-hands-up.jpg"
+blog = ["Elementary school", "Tides Foundation", "project", "classification"]
+shortExcerpt = "We visited our local elementary school and explained the kids how classification works."
```

Comment:
We visited our local elementary school and explained the kids how classification works. -->
We let the fourth grade students build their own decision tree to classify characters into professions.

@janezd ?

Reply:
V bistvu, recimo, ja. Sicer tega res niso delali v Orangeu, tam sem naredil jaz. Ampak so pa naredili na tablo. Tam sem delal tako, kot so predlagali oni. Samo, da sem med njihovimi predlogi izbral tistega, ki mi je bil najbolj všeč. :)

Ampak ja, predlog mi je všeč.

Reply:
Ja, mišljeno je bilo, da so si v glavi sestavili "decision tree". :D Morda damo decision tree dejansko v narekovaje?